<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QWidget" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QRadioButton" name="PatientButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>46</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>患者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>179</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>179</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>89</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>130</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="DoctorButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>46</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>医生</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>89</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>130</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>179</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>DoctorButton</tabstop>
  <tabstop>PatientButton</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>confirm</tabstop>
  <tabstop>RegisterButton</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
